lib/rubocop/cop/rspec/pending_without_reason.rb

```diff
@@ -97,6 +99,14 @@ def on_send(node)
 
         private
 
+        def method_argument?(node)
+          return false unless (send_node = node.ancestors.find(&:send_type?))
```

Comment:
If `pending` is used as an argument, its parent node will be of a send (or csend) type. What is the scenario, in which we should go up more?

Reply:
Ok, I answered myself from the specs: if it's a value in a kwargs / hash

Reply:
Yes, and in the case of array as well.

```console
> ruby-parse -e "is_expected.to match_array [foo, skip, bar]"
(send
  (send nil :is_expected) :to
  (send nil :match_array
    (array
      (send nil :foo)
      (send nil :skip)
      (send nil :bar))))
```

Reply:
Would it match:
```ruby
list = [skip, pending]
foo(list)
```

Is it possible to only trigger the offence if it's directly called?

Would it then detect `pending` inside a conditional?
```ruby
pending if RUBY_VERSION < '3.0'
if RUBY_VERSION < '3.0'
  skip
end
```